pkg/kapp/app/recorded_app.go

```diff
@@ -585,6 +604,19 @@ func (a *RecordedApp) update(doFunc func(*Meta)) error {
 	return nil
 }
 
+func (a *RecordedApp) changeLabelValue() (string, error) {
```

Comment:
Sure, I can make the change. I think I kept it `changeLabelValue` only because the method is on RecordedApp, so it will become `app.appChangeLabelValue` in some places.

Reply:
I would prefer the latter because "app" and "appChange" are two separate values from my perspective 🤔

Reply:
Makes sense.